data.sql

```diff
@@ -5,3 +5,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', '2017-05-12', 5, true, 11);
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Charmander', '2020-02-08',0 , false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Plantmon', '2021-11-15',2 , true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Squirtle', '1993-04-02',3 , false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Angemon', '2005-06-12',1 , true, 45);
```

Comment:
- [ ] Let's pay more attention to the Project requirements 👇🏻 

![Screenshot from 2022-12-07 21-21-17](https://user-images.githubusercontent.com/79058364/206275950-f89fed67-faf2-4943-b1fc-a63fe074f05f.png)

- As you can see above 🙃 , these four requirements have a negative weight value. please fix this 👍🏻